src/Tribe/Aggregator/Tabs/New.php

```diff
@@ -466,18 +466,14 @@ public function maybe_display_aggregator_upsell() {
 		ob_start();
 		?>
 		<div class="notice inline notice-info tribe-dependent tribe-notice-tribe-missing-aggregator-license" data-ref="tribe-missing-aggregator-license" data-depends="#tribe-ea-field-origin" data-condition-empty>
-			<p>
-				<strong><?php esc_html_e( 'Upgrade to Event Aggregator to unlock access to multiple import sources and automatic imports!', 'the-events-calendar' ); ?></strong></p>
-			<p>
-				<?php echo sprintf(
-						esc_html__( 'With Event Aggregator, you can import events from Facebook, iCalendar, Google, and Meetup in a jiffy. Head over to %1$sTheEventsCalendar.com%2$s to purchase instant access, including a year of premium support, updates, and upgrades.', 'the-events-calendar' ),
-						'<a href="https://theeventscalendar.com/wordpress-event-aggregator/?utm_source=importpage&utm_medium=plugin-tec&utm_campaign=in-app">',
-						'</a>'
-					); ?>
-			</p>
-			<p>
-				<a href="https://theeventscalendar.com/wordpress-event-aggregator/?utm_source=importpage&utm_medium=plugin-tec&utm_campaign=in-app" class="tribe-license-link button button-primary"><?php esc_html_e( 'Buy Event Aggregator Now', 'the-events-calendar' ); ?></a>
-			</p>
+
+			<div class="upsell-banner"><?php echo '<img src="' . esc_url( Tribe__Events__Main::instance()->plugin_url . 'src/resources/images/aggregator/upsell-banner.png' ) . '">'; ?></div>
```

Comment:
We're already in an HTML block here - so rather than open PHP tags and echo out more HTML (ie, the image tag) we'd probably get a readability win by only dropping into PHP to echo URL for the `src` attribute.